buildSrc/src/main/kotlin/pklFatJar.gradle.kts

```diff
@@ -40,8 +40,6 @@ val relocations =
   mapOf(
     // pkl-core dependencies
     "org.antlr.v4." to "org.pkl.thirdparty.antlr.v4.",
-    "com.oracle.truffle" to "org.pkl.thirdparty.truffle",
-    "org.graalvm." to "org.pkl.thirdparty.graalvm.",
```

Comment:
this was required, as well as:
```kotlin
val nonRelocations = listOf("com/oracle/truffle/", "org/graalvm/")
```
to fix the JNI issue experienced by `pkl-gradle`. Discussion on Discord thread